src/BoundaryConditions/perturbation_advection_open_boundary_scheme.jl

```diff
@@ -4,6 +4,10 @@ using Oceananigans: defaults
 """
     PerturbationAdvection
 
+Create a `PerturbationAdvection` scheme to be used with an `OpenBoundaryCondition`.
+This scheme will nudge the boundary velocity to the OpenBoundaryCondition's exterior value `val`,
+using a time-scale `inflow_timescale` for inflow and `outflow_timescale` for outflow.
+
```

Comment:
Should we move the whole docstring on the constructor rather than the type?

Reply:
Yes! Done